packs/rabbitmq/sensors/queues_sensor.py

```diff
@@ -0,0 +1,46 @@
+from st2reactor.sensor.base import PollingSensor
+import pika
```

Comment:
Just curious, why did you settle on pika? We use kombu ourselves and debating a move to Pika. Would love to hear your thoughts.